failpoints_test.go

```diff
@@ -1,158 +1,171 @@
+	require.Equal(t, failpoint.ErrNotAllowed, errors.Cause(err))
+	require.Nil(t, val)
 
 	points := map[string]struct{}{}
 	for _, fp := range fps.List() {
 		points[fp] = struct{}{}
 	}
-	c.Assert(points, HasKey, "failpoints-test-1")
-	c.Assert(points, HasKey, "failpoints-test-2")
-	c.Assert(points, HasKey, "failpoints-test-3")
-	c.Assert(points, HasKey, "failpoints-test-4")
-	c.Assert(points, HasKey, "failpoints-test-5")
+	require.Contains(t, points, "failpoints-test-1")
+	require.Contains(t, points, "failpoints-test-2")
+	require.Contains(t, points, "failpoints-test-3")
+	require.Contains(t, points, "failpoints-test-4")
+	require.Contains(t, points, "failpoints-test-5")
 
 	err = fps.Enable("failpoints-test-6", "50*return(5)->1*return(true)->1*return(false)->10*return(20)")
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	// 50*return(5)
 	for i := 0; i < 50; i++ {
 		val, err = fps.Eval("failpoints-test-6")
-		c.Assert(err, IsNil)
-		c.Assert(val.(int), Equals, 5)
+		require.NoError(t, err)
+		require.Equal(t, 5, val.(int))
 	}
 	// 1*return(true)
 	val, err = fps.Eval("failpoints-test-6")
-	c.Assert(err, IsNil)
-	c.Assert(val.(bool), IsTrue)
+	require.NoError(t, err)
+	require.True(t, val.(bool))
 	// 1*return(false)
 	val, err = fps.Eval("failpoints-test-6")
-	c.Assert(err, IsNil)
-	c.Assert(val.(bool), IsFalse)
+	require.NoError(t, err)
+	require.False(t, val.(bool))
 	// 10*return(20)
 	for i := 0; i < 10; i++ {
 		val, err = fps.Eval("failpoints-test-6")
-		c.Assert(err, IsNil)
-		c.Assert(val.(int), Equals, 20)
+		require.NoError(t, err)
+		require.Equal(t, 20, val.(int))
 	}
 	val, err = fps.Eval("failpoints-test-6")
-	c.Assert(errors.Cause(err), Equals, failpoint.ErrNotAllowed)
-	c.Assert(val, IsNil)
+	require.Equal(t, failpoint.ErrNotAllowed, errors.Cause(err))
+	require.Nil(t, val)
 
 	val, err = fps.Eval("failpoints-test-7")
-	c.Assert(errors.Cause(err), Equals, failpoint.ErrNotExist)
-	c.Assert(val, IsNil)
+	require.Equal(t, failpoint.ErrNotExist, errors.Cause(err))
+	require.Nil(t, val)
 
 	val, err = failpoint.Eval("failpoint-env1")
-	c.Assert(err, IsNil)
-	c.Assert(val.(int), Equals, 10)
+	require.NoError(t, err)
+	require.Equal(t, 10, val.(int))
 	val, err = failpoint.Eval("failpoint-env2")
-	c.Assert(err, IsNil)
-	c.Assert(val.(bool), IsTrue)
+	require.NoError(t, err)
+	require.True(t, val.(bool))
 
 	// Tests for sleep
 	ch = make(chan struct{})
 	go func() {
 		defer close(ch)
 		time.Sleep(time.Second)
 		err := failpoint.Disable("gofail/test-sleep")
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 	}()
 	err = failpoint.Enable("gofail/test-sleep", "sleep(100)")
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	start = time.Now()
 	v, err = failpoint.Eval("gofail/test-sleep")
-	c.Assert(err, IsNil)
-	c.Assert(v, IsNil)
-	c.Assert(time.Since(start), GreaterEqual, 90*time.Millisecond, Commentf("not sleep"))
+	require.NoError(t, err)
+	require.Nil(t, v)
+	require.True(t, time.Since(start) > 90*time.Millisecond)
```

Comment:
```suggestion
	require.GreaterOrEqual(t, time.Since(start).Milliseconds(), int64(90))
```

(stretchr/testify#780 😒)

(the original code uses `>=` don't change it to `>`.)